packages/wallet-sdk/src/relay/WalletLinkRelay.ts

```diff
@@ -863,7 +863,7 @@ export class WalletLinkRelay extends WalletSDKRelayAbstract {
   // WIP
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
   // @ts-ignore
```

Comment:
Why is this ignored? have we not added connectAndSignIn to the relay abstract?

Reply:
it's not implemented yet.
it gives error since it's private and not being used.